libr/io/p/io_debug.c

```diff
@@ -96,8 +96,10 @@ static int setup_tokens() {
 }
 
 static int fork_and_ptraceme(RIO *io, int bits, const char *cmd) {
-	PROCESS_INFORMATION pi;
+	PROCESS_INFORMATION pi = { 0 };
 	STARTUPINFOA si = { sizeof (si) };
```

Comment:
In that case this makes no sense. Here you set all the members to "sizeof(si)" right?